### DISCUSSION TITLE
急！参考文献-会议格式

### DISCUSSION THREAD

**Discussion by Bennett561** [sjtug/SJTUThesis#745] [提问] [answered]
latex生成的是这样的，有两个slash ![image](https://user-images.githubusercontent.com/17747051/147044572-5b575892-bd81-4e1e-8311-12dd9546b66e.png)
而我们学院提供的模板里没有这个，
![image](https://user-images.githubusercontent.com/17747051/147044746-5a68257f-e6a9-4c77-bfa4-789d2de1e1b3.png)
这个请问怎么改呀

**Comment by zepinglee** [comment]
第一个示例是会议论文，第二个是会议的论文集，两者不一样。

**Reply by Bennett561** [reply]
那咱们的模板有办法统一为下面的格式吗……

**Reply by AlexaraWu** [reply]
条目的类型不一样。

有斜杠的是论文集中的论文，类型是 `inproceedings` 或者 `incollection` ，格式是 "论文标题[C]//论文集标题"；

没有斜杠的是整个论文集，类型是 `proceedings` 或者 `collection` 。

**Reply by Bennett561** [reply]
噢噢懂了，多谢

**Comment by hushidong** [comment, marked as answer]
要改也很方便，映射一下就可以。比如，加上：

\DeclareBibliographyAlias{inproceedings}{proceedings}

比如：
```
\documentclass{ctexart}
\usepackage[paperheight=16cm]{geometry}
\usepackage[style=gb7714-2015]{biblatex}
\usepackage[colorlinks]{hyperref}

\makeatletter

%    \DeclareStyleSourcemap{
%        \maps[datatype=bibtex]{
%            \map{
%            \step[typesource=inproceedings, typetarget=Proceedings, final]
%            }
%        }
%    }

\DeclareBibliographyAlias{inproceedings}{proceedings}

\makeatother

\begin{filecontents}{\jobname.bib}
@InProceedings{王临慧2010-138,
  author    = {王临慧},
  title     = {从几组声母的演变看天津方言形成的自然条件和历史条件},
  bookauthor= {曹志耘},
  booktitle = {汉语方言的地理语言学研究：首届中国地理语言学国际学术研讨会论文集},
  year      = {2010},
  publisher = {北京语言大学出版社},
  pages     = {138},
  address   = {北京},
}

@Proceedings{牛志明2012--,
  title     = {综合湿地管理国际研讨会论文集},
  year      = {2012},
  editor    = {牛志明 and 斯温兰德 and 雷光春},
  publisher = {海洋出版社},
  address   = {北京},
}

\end{filecontents}

\addbibresource{\jobname.bib}

\begin{document}

\nocite{*}
\printbibliography
\end{document} 
```

**Reply by Bennett561** [reply]
嗷嗷，懂了，就是把inproceedings看作是proceedings

**Reply by wy12318** [reply]
原因是这个：![微信图片_20241226224534](https://github.com/user-attachments/assets/5a271511-10ec-45d2-ade9-a095c91bdf56)
在setup.tex中添加
```\ExecuteBibliographyOptions{gbpunctin=false}```即可
